src/helpers/component-tree.ts

```diff
@@ -1,10 +1,14 @@
 import { ReactTestInstance } from 'react-test-renderer';
 
+export type HostReactTestInstance = ReactTestInstance & { type: string };
```

Comment:
We probably need to export this type as well in `pure.js`

Reply:
Oh right, good catch!